packages/components/inline-message/src/inline-message.scss

```
@@ -0,0 +1,150 @@
+$statuses: info, success, warning, danger;
+
+:host {
+  --_background: var(--_background-color);
+  --_border: var(--sl-border-width-inline-message-default) solid var(--_border-color);
+  --_border-left: var(--sl-border-width-inline-message-start) solid var(--_border-color);
+  --_border-radius: var(--sl-border-radius-inline-message-default);
+  --_padding-block: var(--sl-space-inline-message-block);
+  --_padding-inline-start: calc(var(--sl-space-inline-message-inline-start));
+  --_padding-inline-end: var(--sl-space-inline-message-inline-end);
```

```suggestion
  --_padding-inline: var(--sl-space-inline-message-inline-start) var(--sl-space-inline-message-inline-end);
```
so you online have 1 variable you need to set later on.